packages/tools/tests/test/visualization/config.json

```diff
@@ -1371,6 +1371,19 @@
             "playgroundId": "#BCYE7J#31",
             "referenceImage": "multiCamerasOutputRenderTarget.png"
         },
+        {
+            "title": "Multi cameras and output render target with depth stencil texture (D24_S8)",
+            "renderCount": 2,
+            "playgroundId": "#BCYE7J#33",

```

Comment:
This PG already works without the PR: the fact that it is using only a depth and not a depth/stencil texture can't be seen (we can't read the stencil in WebGL2). I don't think it's possible to check that the right format is really used under the hood, so it's probably better to remove this validation test.

Reply:
I simplified the case down to just one which exercises a GL error prior to my change that would lead to a blank render target.

https://www.babylonjs-playground.com/#BCYE7J#36